mlir/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ list(INSERT CMAKE_MODULE_PATH 0
   )
 
 include(AddMLIR)
+include(HandleMLIROptions)
 
 # -BSymbolic is incompatible with TypeID
 if("${CMAKE_SHARED_LINKER_FLAGS}" MATCHES "-Bsymbolic[^-]")
```

Comment:
There are a bunch of thing right below here that seem not conceptually different from you addition, so why should all these be in different files?